### PR TITLE
Fixes #480 getAllowedCores returns integer or NULL if <1

### LINKED ISSUE #480: TestThat test faield on ValidR LINUX environment.
Two test cases from test that fail, after installation on ValidR. From the log's I can not tell what fails, and how to check what the problem is.

Could you help with this?

[1] "05/05/2021 - 11:48:59 : Starting run of population workflow"
[1] "05/05/2021 - 11:48:59 : Starting Perform simulation task"
[1] "05/05/2021 - 11:48:59 : Run simulation in simulation set: A"
[1] "05/05/2021 - 11:48:59 : test_check: argument 'value' is of type 'numeric', but expected 'integer'!"
── 1. Error: (unknown) (@test-pop-gof.R#115)  ──────────────────────────────────
test_check: argument 'value' is of type 'numeric', but expected 'integer'!
Backtrace:
 1. workflowNoObs1$runWorkflow()
 5. ospsuite.reportingengine:::validateIsInteger(value)
 6. ospsuite.reportingengine:::logErrorThenStop(...)

[1] "05/05/2021 - 11:49:20 : Starting run of population workflow"
[1] "05/05/2021 - 11:49:20 : Starting Perform simulation task"
[1] "05/05/2021 - 11:49:20 : Run simulation in simulation set: Adults"
[1] "05/05/2021 - 11:49:31 : test_check: argument 'value' is of type 'numeric', but expected 'integer'!"
── 2. Error: (unknown) (@test-pop-pk-parameters.R#82)  ─────────────────────────
test_check: argument 'value' is of type 'numeric', but expected 'integer'!
Backtrace:
 1. workflowPediatric$runWorkflow()
 5. ospsuite.reportingengine:::validateIsInteger(value)
 6. ospsuite.reportingengine:::logErrorThenStop(...)

[ospsuite.reportingengine_results.zip](https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/files/6459979/ospsuite.reportingengine_results.zip)

### PR DESCRIPTION
**Pull request by pchelle**

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -290,7 +290,9 @@ getAllowedCores <- function() {
     # get cpu allowance from files
     cfs_quota_us <- as.numeric(system("cat /sys/fs/cgroup/cpu/cpu.cfs_quota_us", intern = T))
     cfs_period_us <- as.numeric(system("cat /sys/fs/cgroup/cpu/cpu.cfs_period_us", intern = T))
-    cfs_quota_us / cfs_period_us
+    cores <- floor(cfs_quota_us / cfs_period_us)
+    if(cores < 1){return()}
```
Comment (Yuri05) [inline R/utils.R:294]: is `return()` the same as `return(NULL)` ?

Comment (pchelle) [reply R/utils.R:294]: Yes.

Comment (msevestre) [reply R/utils.R:294]: can we have return on another line please
Also run the styler

**Review by msevestre — CHANGES_REQUESTED** [review]

File: R/utils.R [hunk 2]
```diff
@@ -290,7 +290,11 @@ getAllowedCores <- function() {
     # get cpu allowance from files
     cfs_quota_us <- as.numeric(system("cat /sys/fs/cgroup/cpu/cpu.cfs_quota_us", intern = T))
     cfs_period_us <- as.numeric(system("cat /sys/fs/cgroup/cpu/cpu.cfs_period_us", intern = T))
-    cfs_quota_us / cfs_period_us
+    cores <- floor(cfs_quota_us / cfs_period_us)
+    if (cores < 1) {
+      return()
```
Comment (msevestre) [inline R/utils.R:295]: let's use return(NULL) because two lines under, we use it explicitly.

**Review by msevestre — CHANGES_REQUESTED** [review]
sorry missed this one

**Pull request merged**